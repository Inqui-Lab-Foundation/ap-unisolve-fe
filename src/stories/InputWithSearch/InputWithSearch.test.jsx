/* eslint-disable no-undef */
// import {fireEvent, render, screen} from "@testing-library/react";
import renderer from 'react-test-renderer';
// import React from 'react';

// import InputWithSearchComp from "./InputWithSearch";
//https://stackoverflow.com/questions/41991077/testing-react-select-component


it('checkButtonRender' ,() => {
    const component = renderer.create(`<InputWithSearchComp />`);
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});

// describe('changeInput',() => {
//     it('onChange',() => {
//         const component = renderer.create(`<InputWithSearchComp />`)
//         let tree = component.toJSON();
//     fireEvent.change(component,{target:{value:"letscode"}})
//     expect(component.value).toBe('letscode')
//     })
// })