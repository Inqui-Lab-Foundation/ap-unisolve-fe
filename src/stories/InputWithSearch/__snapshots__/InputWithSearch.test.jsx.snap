// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checkButtonRender 1`] = `"<InputWithSearchComp />"`;
